rasa/core/channels/botframework.py

```diff
@@ -214,6 +214,14 @@ async def health(request: Request) -> HTTPResponse:
 
         @botframework_webhook.route("/webhook", methods=["POST"])
         async def webhook(request: Request) -> HTTPResponse:
+            """ Webhook receiving requests from MS Botframework.
+            
```

Comment:
oups, still some whitespaces left here:

```suggestion
            """Webhook receiving requests from MS Botframework.

```